arcsi/templates/security/login_user.html

```diff
@@ -9,6 +9,7 @@ <h1>{{ _('Login') }}</h1>
     {{ render_field_with_errors(login_user_form.email) }}
     {{ render_field_with_errors(login_user_form.password) }}
     {{ render_field_with_errors(login_user_form.remember) }}
```

Comment:
This will conflict on the master because we merged this commit previously:
https://github.com/lahmacunradio/arcsi/commit/26d66f1f6eaf27ec950564351ed22916143b9109#diff-3f943558434bd09c516da347490e7daae691dad562b7162add1fe90f03931163R15